respec/template.html

```diff
@@ -436,13 +434,101 @@ <h2>Conformance</h2>
   </p>
 </section>
 
+<section>
+  <h2>References</h2>
```

Comment:
See respec docs: https://respec.org/docs/#references-0

Reply:
OK I see, but still think it's worth keeping the References section now I added it, to make it easier to get into regular OMG format. I did also beef up the Namespaces section for some that were missing

Reply:
Yeah, but not at the top? Readers then first have to wade through too much noise before getting to the meat